Paws4Me/Paws4Me/Display All Adoptable Pets/View Controllers/AllPetDetailViewController.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  AdoptPet.swift
+//  Paws4Me
+//
+//  Created by Kirsty-lee Walker 20/02/2022
+//
+
+import UIKit
+
+class AllPetDetailViewController: UIViewController, UISearchBarDelegate {
```

Comment:
Let's conform to `UISearchBarDelegate` in an extension.

Reply:
Done 👍🏻